core/eolearn/tests/test_core_tasks.py

```diff
@@ -71,38 +72,39 @@ def patch_fixture():
     return patch
```

Comment:
BTW, I noticed there is already an existing fixture for core tests [here](https://github.com/sentinel-hub/eo-learn/blob/feat/rename-timestamp/core/eolearn/tests/conftest.py#L20). Any idea why a dedicated fixture is used here?

Reply:
`TestEOPatch`is kinda big and loading it 40 times should be that quick but i might be wrong going to "time" test it after other refactoring if this is ok ?

Reply:
In the long run it would be great if we get rid of the test eopatch as it is a lot of data on the repo.
Here we don't much care about what data is in, only that it is populated. Although it could be improved a bit (some floats, some random etc.)

What i would like to point out is that this EOPatch only has a single feature per type, which should be adjusted a bit. This can be done manually for the sake of this MR, but in the long run we would want a utility for generating 'random-ish' patches.

Reply:
Adjusted a bit.